script/openqa-clone-job

```diff
@@ -215,10 +220,11 @@ sub parse_options () {
         "skip-deps", "skip-chained-deps", "skip-download", "parental-inheritance",
         "help|h", "show-progress", "within-instance|w=s", "clone-children",
         "max-depth:i", "repeat|r:i", "ignore-missing-assets", "export-command",
-        "skip-checks",
+        "skip-checks","check-repos",
     ) or usage(1);
     usage(0) if $options{help};
     usage(1) if $options{help} || ($options{'within-instance'} && $options{from});
+    print STDERR "--skip-checks is deprecated and will be dropped soon. Just remove it because checks not executed by default anymore" if $options{'skip-checks'};
```

Comment:
```suggestion
    print STDERR "--skip-checks is deprecated and will be dropped soon. Just remove it because checks are not executed by default anymore" if $options{'skip-checks'};
```